test-count.ps1

```diff
@@ -0,0 +1,39 @@
+$e2ePath = "Tests/E2E"
+$integrationPath = "Tests/Integration" 
+$unitPath = "Tests/Unit"
+
+$testAttributeRegex = "\[ *(Fact|Theory) *\]"
+
+$outputFile = "test-count.csv"
+$fullOutputPath = Join-Path -Path $PWD -ChildPath $outputFile
+$gitRepoPath = "C:\sources\lora\iotedge-lorawan"
+
+Write-Host "Current directory: " $PWD " OutputPath: " $fullOutputPath
+Clear-Content -Path $fullOutputPath -ErrorAction SilentlyContinue
+Push-Location $gitRepoPath
```

Comment:
If you're going to change the location, I would put a try-finally around the rest of the script and do the `Pop-Location` in the finally part so that the location is restored in the event of any error.